assets/src/js/commands/admin-commands.js

```diff
@@ -0,0 +1,179 @@
+		},
+		{
+			name: 'new-post-type',
+			label: __( 'Create New Post Type', 'secure-custom-fields' ),
+			url: `${ adminUrl }post-new.php?post_type=acf-post-type`,
+			icon: 'plus',
+			description: __(
+				'SCF: Create a new custom post type',
+				'secure-custom-fields'
+			),
+			keywords: [ 'add', 'new', 'create', 'cpt', 'content type' ],
+		},
+		{
+			name: 'taxonomies',
+			label: __( 'Taxonomies', 'secure-custom-fields' ),
+			url: `${ adminUrl }edit.php?post_type=acf-taxonomy`,
+			icon: 'category',
+			description: __(
+				'SCF: Manage custom taxonomies for organizing content',
+				'secure-custom-fields'
+			),
+			keywords: [ 'categories', 'tags', 'terms', 'custom taxonomies' ],
+		},
+		{
+			name: 'new-taxonomy',
+			label: __( 'Create New Taxonomy', 'secure-custom-fields' ),
+			url: `${ adminUrl }post-new.php?post_type=acf-taxonomy`,
+			icon: 'plus',
+			description: __(
+				'SCF: Create a new custom taxonomy',
+				'secure-custom-fields'
+			),
+			keywords: [
+				'add',
+				'new',
+				'create',
+				'taxonomy',
+				'categories',
+				'tags',
+			],
+		},
+		{
+			name: 'options-pages',
+			label: __( 'Options Pages', 'secure-custom-fields' ),
+			url: `${ adminUrl }edit.php?post_type=acf-ui-options-page`,
+			icon: 'admin-settings',
+			description: __(
+				'SCF: Manage custom options pages for global settings',
+				'secure-custom-fields'
+			),
+			keywords: [ 'settings', 'global options', 'site options' ],
+		},
+		{
+			name: 'new-options-page',
+			label: __( 'Create New Options Page', 'secure-custom-fields' ),
+			url: `${ adminUrl }post-new.php?post_type=acf-ui-options-page`,
+			icon: 'plus',
+			description: __(
+				'SCF: Create a new custom options page',
+				'secure-custom-fields'
+			),
+			keywords: [ 'add', 'new', 'create', 'options', 'settings page' ],
+		},
+		{
+			name: 'tools',
+			label: __( 'SCF Tools', 'secure-custom-fields' ),
+			url: `${ adminUrl }admin.php?page=acf-tools`,
+			icon: 'admin-tools',
+			description: __(
+				'SCF: Access SCF utility tools',
+				'secure-custom-fields'
+			),
+			keywords: [ 'utilities', 'import export', 'json' ],
+		},
+		{
+			name: 'import',
+			label: __( 'Import SCF Data', 'secure-custom-fields' ),
+			url: `${ adminUrl }admin.php?page=acf-tools&tool=import`,
+			icon: 'upload',
+			description: __(
+				'SCF: Import field groups, post types, taxonomies, and options pages',
+				'secure-custom-fields'
+			),
+			keywords: [ 'upload', 'json', 'migration', 'transfer' ],
+		},
+		{
+			name: 'export',
+			label: __( 'Export SCF Data', 'secure-custom-fields' ),
+			url: `${ adminUrl }admin.php?page=acf-tools&tool=export`,
+			icon: 'download',
+			description: __(
+				'SCF: Export field groups, post types, taxonomies, and options pages',
+				'secure-custom-fields'
+			),
+			keywords: [ 'download', 'json', 'backup', 'migration' ],
+		},
+	];
+
+	commands.forEach( ( command ) => {
+		commandStore.registerCommand( {
+			name: 'acf/' + command.name,
```

Comment:
Nit: What do you think about the following?
```suggestion
			name: 'scf/' + command.name,
```

Reply:
Yes, that should be the right prefix. I'll fix it for the CPT commands as well.